generated/nidaqmx/_lib.py

```diff
@@ -6,6 +6,9 @@
 import platform
 import sys
 import threading
+import locale
+import os
+from dotenv import load_dotenv
```

Comment:
As far as I know, `nidaqmx` doesn't depend on `dotenv`. If it does, that's purely by accident.

If this worked, it's because this project's dev dependencies happened to depend on `dotenv`. This code may break when users install `nidaqmx` in their own projects.